modules/openapi-generator/src/main/resources/typescript/api/api.mustache

```diff
@@ -43,7 +46,10 @@ export class {{classname}}RequestFactory extends BaseAPIRequestFactory {
      {{/allParams}}
      */
     public async {{nickname}}({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}_options?: Configuration): Promise<RequestContext> {
-        let _config = _options || this.configuration;
+	let _config = this.configuration;
```

Comment:
so this is changing the priority, is this intentional?

Reply:
reverted